Game/Source/GDKTestGyms/BenchmarkGymGameModeBase.cpp

```diff
@@ -49,6 +49,9 @@ namespace
 	
 	const FString NumWorkersWorkerFlag = TEXT("num_workers");
 	const FString NumWorkersCommandLineKey = TEXT("-NumWorkers=");
+
+	const FString NumSpawnZonesWorkerFlag = TEXT("num_spawn_zones");
+	const FString NumSpawnZonesCommandLineKey = TEXT("-NumSpawnZones=");
```

Comment:
Maybe just NumZones would be enough? We don't use that terminology for offloading really

Reply:
Well this was just to separate it from multi worker completely. Now you just ask it to change how the spawn clusters are spawned. If there is a better way to encapsulate this I am happy to change it. NumZones ties it into multi worker IMO.